### ISSUE TITLE
parent/child mapping support?

### ISSUE THREAD

**Issue by hjz** [codelibs/elasticsearch-reindexing#6]
Does this support setting the parentId on a child document?

**Comment by marevol** [comment]
Reindexing plugin does not copy an index setting/mapping.
If you have parent-child docs in the source index, please create the target index with setting/mapping before starting reindexing.

**Comment by hjz** [comment]
Right, we'll recreate the index. As it's reindexing documents, does it also call setParent when required?

**Comment by marevol** [comment]
Hmm, I see... It's not support at the moment. I would like to find out about the solution.

**Comment by hjz** [comment]
Let me know what you find. I'd be great to have support for this.

**Comment by marevol** [comment]
Fixed at #7
